airbyte_cdk/cli/airbyte_cdk/exceptions.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+"""Exceptions for the Airbyte CDK CLI."""
+
+from dataclasses import dataclass
+
+from airbyte_cdk.sql.exceptions import AirbyteConnectorError
+
+
+@dataclass(kw_only=True)
+class ConnectorSecretWithNoValidVersionsError(AirbyteConnectorError):
```

Comment:
I'm removing the inheritance from `AirbyteConnectorError`.

```suggestion

@dataclass(kw_only=True)
class ConnectorSecretWithNoValidVersionsError(Exception):
```